lib/structs/project_details.py

```diff
@@ -8,7 +8,6 @@ class ProjectDetails:
     name: str
     email: str
     description: str = ""
-    project_domain: str = None
```

Comment:
I thought `project_domain` is used to determine which LDAP server they are authenticating against?
Without this it will use the STFC LDAP by default (unless I misunderstand how keystone was wired up)

Reply:
Answered in the commit msg for https://github.com/stfc/st2-cloud-pack/pull/354/commits/99202c635c004bd5be52e8f8a7c66c6c4f4ab2e5